.github/workflows/build.yml

```diff
@@ -124,6 +124,14 @@ jobs:
           echo "TAP version 13\r" > git.tap
```

Comment:
what I was trying to say is that raising from 2.19 to 2.20 shouldn't reduce the "out-of-the-box support for those distros", since they either package something >= 2.20 or < 2.19.

Reply:
Hm. Okay, I think see what you mean now: this bump in the required version will not change the current partition of "supported/unsupported distros".